samples/ravendb/simple/Raven_8/Client/Program.cs

```diff
@@ -1,44 +1,37 @@
 using System;
-using System.Threading.Tasks;
 using NServiceBus;
 
-class Program
-{
-    static async Task Main()
-    {
-        Console.Title = "Samples.RavenDB.Client";
-        var endpointConfiguration = new EndpointConfiguration("Samples.RavenDB.Client");
-        endpointConfiguration.EnableInstallers();
-        endpointConfiguration.UseSerialization<SystemJsonSerializer>();
-        endpointConfiguration.UseTransport(new LearningTransport());
-
-        var endpointInstance = await Endpoint.Start(endpointConfiguration)
-            .ConfigureAwait(false);
+Console.Title = "Samples.RavenDB.Client";
+var endpointConfiguration = new EndpointConfiguration("Samples.RavenDB.Client");
+endpointConfiguration.EnableInstallers();
```

Comment:
Nitpick: I know this isn't needed when the learningtransport is used but it might be that RavenDB somehow needs it